src/schedule/zephyr_domain.c

```diff
@@ -78,8 +78,11 @@ static void zephyr_domain_thread_fn(void *p1, void *p2, void *p3)
 			diff = UINT32_MAX - cycles0 + cycles1;
 
 		timer_fired = k_timer_status_get(&zephyr_domain->timer);
-		if (timer_fired > 1)
+		if (timer_fired > 1) {
 			overruns++;
+			tr_warn(&ll_tr, "LL timer overrun: took %u cycles, missed %u interrupts",
+				diff, timer_fired);
+		}
```

Comment:
Not sure about this.  The system is under a load spike, so adding an immediate warning to the mix is likely to make the situation worse.

Especially now as the IPC handling can cause spikes to LL processing time (there is work ongoing to address this, e.g. what @aborisovich is doing), but until then, I'd rather report overruns via the periodic report.